server/src/main/java/com/linecorp/centraldogma/server/storage/project/InternalProjectInitializer.java

```diff
@@ -56,12 +57,26 @@ public InternalProjectInitializer(CommandExecutor executor) {
     }
 
     /**
-     * Creates an internal project and repositories such as a token storage.
+     * Creates an internal project and repositories and a token storage to {@code dogma/dogma/tokens.json}.
      */
     public void initialize() {
+        initialize(INTERNAL_PROJECT_DOGMA, true);
+    }
+
+    /**
+     * Creates the specified internal project and its internal repositories.
+     */
+    public void initialize(String projectName) {
+        requireNonNull(projectName, "projectName");
+        checkArgument(!INTERNAL_PROJECT_DOGMA.equals(projectName),
+                      "Use initialize() to create %s", projectName);
+        initialize(projectName, false);
+    }
+
+    private void initialize(String projectName, boolean createTokenFile) {
```

Comment:
Optional) I'm aware this is internal, but instead of a flag that doesn't really have a meaning would just defining a `initializeProject` method make more sense?

```java
    public void initialize() {
        initializeProject(INTERNAL_PROJECT_DOGMA);
        initializeTokens();
    }

    public void initializeProject(String projectName) {
        ...
```